edelphi/src/main/resources/fi/metatavu/edelphi/liquibase/subscription-level-features.sql

```diff
@@ -0,0 +1,40 @@
+INSERT INTO SubscriptionLevelFeature (id, feature, subscriptionLevel) VALUES 
+  (1, 'BASIC_USAGE', 'PREMIUM'),
+  (2, 'CREATE_PANELS', 'PREMIUM'),
+  (3, 'ACCESS_HELPDESK', 'PREMIUM'),
+  (4, 'MANAGE_PANEL_QUERIES', 'PREMIUM'),
+  (5, 'MANAGE_PANEL', 'PREMIUM'),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal 'MANAGE_PANEL' 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=plsql%3AS1192)